app/actions/local/thread.ts

```diff
@@ -221,6 +222,7 @@ export async function processReceivedThreads(serverUrl: string, threads: Thread[
                     users.push(participant);
                 }
             });
+            threadsToHandle.push({...threads[i], lastFetchedAt: post.create_at});
```

Comment:
shouldn't you be using max as we discussed?

Reply:
This is when we receive the threads, not the threads posts. If I understood correctly, the post you see here is the root post. So whenever we fetch the posts of this thread, we want all the posts after the creation of the first post. If we were to use the max here, if the initial post of the thread gets updated, we would only get the posts after the update, and we will see a gap.